---
title: "dseqr-data"
output: rmarkdown::html_vignette
vignette: >
  %\VignetteIndexEntry{dseqr-data}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>"
)
```

```{r setup}
library(dseqr.data)
```

# Installation

`dseqr.data` can be installed from Bioconductor as follows:

```{r, eval = FALSE}
if(!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install("dseqr.data")
```

# Overview of dseqr.data

`dseqr.data` includes:
  - experiment data from CMAP02 and L1000 for the `dseqr` package
  - annotation used by `rkal` for importing RNA-Seq data

# Getting Started using dseqr.data

To load and download CMAP02 and L1000 data, `dseqr` does the following:


```{r}
# to load the actual data replace 
# 'example.rds' with e.g. 'l1000_genes_es.rds'
drug_es <- load_drug_es('example.rds')
```

To generate transcript-to-gene maps for importing the results of `kallisto`
quantification, `rkal` calls the following:

```{r}
tx2gene <- load_tx2gene('Homo sapiens', release='94')
```

# Session info

The following package and versions were used in the production of this vignette.

```{r echo=FALSE}
sessionInfo()
```


 
